libfwupdplugin/fu-quirks.h

```diff
@@ -430,3 +430,17 @@ fu_quirks_add_possible_key(FuQuirks *self, const gchar *possible_key);
  * Since: 1.8.2
  **/
 #define FU_QUIRKS_CFI_DEVICE_BLOCK_SIZE "CfiDeviceBlockSize"
+/**
+ * FU_QUIRKS_SHOW_GUID
```

Comment:
Doesn't that need to mention somewhere "true by default" and/or should be explicitly set to "false"?

Reply:
I could flip it around -- i.e. make the key `HideGuid=true` -- @superm1 any better names? `InstanceIdOnlyQuirk=true` perhaps?

Reply:
mmm... how about `HideGenericGuid` literally?

Reply:
How about `ShowOnlyPluginGuids`?

Reply:
> ShowOnlyPluginGuids

Nah, we're only hiding the specific one that we match in the quirk file. @d4s is right that it's the wrong way around, so I just flipped it around to `HideGuid`.